docs/text/parallelization.rst

```diff
@@ -29,31 +29,8 @@ Parallelization of Feature Extraction
 
 For the feature extraction tsfresh exposes the parameters
 `n_processes` and
-`chunksize`. Both behave anlogue to the parameters
+`chunksize`. Both behave analogue to the parameters
 for the feature selection.
 
-Additionally there are two options for how the parallelization is done:
-
-1.  ``'per_kind'`` parallelizes the feature calculation per kind of time series.
-2.  ``'per_sample'`` parallelizes per kind and per sample.
-
-To enforce an option, either pass ``'per_kind'`` or ``'per_sample'`` as the ``parallelization=`` parameter of the
-:func:`tsfresh.extract_features` function. By default the option is chosen with a rule of thumb:
-
-If the number of different time series (kinds) is less than half of the number of available worker
-processes (``n_processes``) then ``'per_sample'`` is chosen, otherwise ``'per_kind'``.
-
-Generally, there is no perfect setting for all cases. On the one hand more parallelization can speed up the calculation
-as the work is better distributed among the computers resources. On the other hand parallelization
-introduces overheads such as copying data to the worker processes, splitting the data to enable the distribution and
-combining the results.
-
-Implementing the parallelization we observed the following aspects:
-
--   For small data sets the difference between parallelization per kind or per sample should be negligible.
--   For data sets with one kind of time series parallelization per sample results in a decent speed up that grows
-    with the number of samples.
--   The more kinds of time series the data set contains, the more samples are necessary to make parallelization
-    per sample worthwhile.
--   If the data set contains more kinds of time series than available cpu cores, parallelization per kind is
-    the way to go.
+To do performance studies and profiling, it sometimes quite useful to turn off parallelization at all. This can be
+handled by the boolean parameter `parallelization`.
```

Comment:
Yes, seems reasonable. I would actually prefer to use `n_jobs = 0` for no parallelisation, as one could test the overhead of parallelisation when comparing `n_jobs = 0` and `n_jobs = 1`.